bitstream.h

```diff
@@ -34,21 +34,25 @@ class cBitstream
   int      m_offset;
   int      m_len;
   bool     m_error;
+  bool     m_doEP3;
```

Comment:
That was my first though, but I wasn't sure if I don't override readGolombUE too, would a call to readGolombUE call the cBitstreamEP3::readBits  or cBitstream::readBits, so I duplicated the functions needed in order to leave original code untouched. 

Then, in the second commit, I wanted to do only the bitstream optimization as requested. But in any case, that requires a change in parse_hevc for the class name, so this change will also be in the commit.